test/libsolidity/InlineAssembly.cpp

```diff
@@ -291,27 +320,27 @@ BOOST_AUTO_TEST_CASE(oversize_string_literals)
 
 BOOST_AUTO_TEST_CASE(assignment_after_tag)
 {
-	BOOST_CHECK(successParse("{ let x := 1 { tag: =: x } }"));
+	BOOST_CHECK(successParse("{ let x := 1 { 7 tag: =: x } }"));
```

Comment:
This is just a very bad example that is mainly used to check that `tag:=:x` is parsed correctly. Note that the inner block does not define x, it just moves the values on top of the stack into the slot named `x`. As I said in earlier meetings, you should not use labels in "clean code" at all.